previews/primer/beta/button_group_preview.rb

```diff
@@ -63,22 +63,18 @@ def icon_buttons(size: :medium)
         end
       end
 
-      # @label Button group with all tags
+      # @label Button group with multiple tags
```

Comment:
I don't know if we'd ever want a `summary` tag in a button group, so I've removed this example.